controller/lib/core/controller.cpp

```diff
@@ -31,11 +33,30 @@ static DebugFloat dbg_blower_valve_ki("blower_valve_ki",
 static DebugFloat dbg_blower_valve_kd("blower_valve_kd",
                                       "Derivative gain for blower valve PID");
 
+// In an ideal world we'd fully close the exhale valve during inhale, so as not
+// to waste any oxygen.  In practice having the valve be somewhat open is
+// helpful for the following reasons.
+//
+//  - If the exhale valve is not open, then (modulo leakage) pressure can never
+//    go down.  This means that if our control overshoots, we can't recover.
+//
+//  - With the blower at full speed, there's some leakage through the inhale
+//    pinch valve even when it's fully closed.  This means that pressure may
+//    continue to increase once we hit PIP -- unless we have an outlet for that
+//    pressure.
+static DebugFloat dbg_exhale_valve_on_pip(
+    "exhale_valve_on_pip",
+    "Position of exhale valve [0 = closed, to 1 = open] during inhale.", 0.3f);
```

Comment:
I guess this 0.3 value will end up dependent on PIP once we have enough test data to determine the correct value for each setting, or should it be good enough as-is?

Reply:
No idea; it might depend only on PIP, it might depend on PIP and lung compliance (ugh), or maybe 0.3 is good enough.  0.3 came from Edwin's tests.

Tomorrow I'm going to write up a script to let us run and capture all of the covent testcases.